packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss

```diff
@@ -34,55 +34,30 @@ $progress-ring-size: var(--progress-ring-size, 24px);
 }
 
 :host .file-list-wrapper {
-  background-color: $background-color;
+  background-color: $file-list-background-color;
   box-shadow: $file-list-box-shadow;
   border-radius: $file-list-border-radius;
   border: $file-list-border;
   position: relative;
-  color: $color;
   display: flex;
   flex-direction: column;
-  border-radius: 8px;
 
   .file-list {
     display: flex;
-    padding: var(--file-list-padding, 4px 0);
-    margin: var(--file-list-margin, 0);
+    padding: $file-list-padding;
+    margin: $file-list-margin;
     flex-direction: column;
     list-style: none;
 
     .file-item {
-      border-top: var(--file-item-border-top, none);
-      border-left: var(--file-item-border-left, none);
-      border-right: var(--file-item-border-right, none);
-      border-bottom: var(--file-item-border-bottom, none);
-      margin: $file-item-margin;
-      padding-inline-start: var(--file-item-padding-inline-start, 24px);
-      padding-top: var(--file-item-padding-top, 10px);
-      padding-bottom: var(--file-item-padding-bottom, 8px);
-      border-radius: $file-item-border-radius;
       cursor: pointer;
+      border-radius: var(--file-border-radius);
+      margin: var(--file-margin, 2px 4px);
 
       .mgt-file-item {
-        --file-background-color: $background-color;
-      }
-
-      &:hover,
-      &.focused {
-        background-color: $file-item-background-color--hover;
-
-        .mgt-file-item {
-          --file-background-color: $file-item-background-color--hover;
-        }
-      }
-
-      &:active,
-      &.selected {
-        background-color: $file-item-background-color--active;
-
-        .mgt-file-item {
-          --file-background-color: $file-item-background-color--active;
-        }
+        --file-padding: 10px 8px 10px 8px;
```

Comment:
does this still leave the opportunity for a developer to set the --file-padding when using an mgt-file-list?

Reply:
Yes. Any mgt-file token can be re-set for mgt-file-list. It will override this value.

Reply:
Also, this is specifically styling for an uploaded file. I haven't found a better way to do customization of the same component, used in different parts of another component/DOM.

Reply:
Line 83-85 is setting the font size on the button that is used to load more items (unless I'm missing something)
The same value is also set on the show-more class as the show-more-text class. So, I'm a bit confused how that relates to an uploaded file and why both font-size rules are needed?

Reply:
Yes, that's confusing because the file changed after I made the comment. What I meant was these lines https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/5915b0276f383058173183e4aaa240d128f1ceff/packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss#L57-L61

I use a class `.mgt-file-item` that is set on the uploaded item `mgt-file` to style that component without affecting the other `mgt-file`s in the file list.

Reply:
Ahh gotcha, and this class is used in both the mgt-file-list and mgt-file-upload to apply different token values.